patch_api/api/estimates_api.py

```diff
@@ -1205,13 +1200,8 @@ def create_rail_shipping_estimate_with_http_info(
         all_params.append("average_daily_balance_btc_sats")
         all_params.append("average_daily_balance_eth_gwei")
         all_params.append("timestamp")
-        all_params.append("destination_country_code")
-        all_params.append("destination_locode")
-        all_params.append("destination_postal_code")
-        all_params.append("freight_mass_g")
-        all_params.append("origin_country_code")
-        all_params.append("origin_locode")
-        all_params.append("origin_postal_code")
+        all_params.append("origin_airport")
+        all_params.append("destination_airport")
```

Comment:
@holtbp

Reply:
Huh, removing all of the destination/origin fields and putting airport fields for this rail shipping estimate method?